CRM/Logging/ReportDetail.php

```diff
@@ -242,4 +235,18 @@ public function buildQuickForm() {
     $this->assign('revertConfirm', ts('Are you sure you want to revert all changes?'));
   }
 
+  /**
+   * Get the properties that might be in the URL.
+   */
+  protected function getPropertiesFromUrl() {
```

Comment:
Nitpick: A function named `getFoo()` should return the value `foo`. In this case, the return is `void` and the results are conveyed by altering state, so a better name would be `loadFoo()` / `loadPropertiesFromUrl()` or maybe `parseFoo()` / `parsePropertiesFromUrl()`